themes/bira.zsh-theme

```diff
@@ -11,11 +11,18 @@ else
     rvm_ruby='%{$fg[red]%}‹$(rbenv version | sed -e "s/ (set.*$//")›%{$reset_color%}'
   fi
 fi
-local git_branch='$(git_prompt_info)%{$reset_color%}'
 
-PROMPT="╭─${user_host} ${current_dir} ${rvm_ruby} ${git_branch}
-╰─%B$%b "
+PROMPT_START="╭ ${user_host} %t ${current_dir} ${rvm_ruby}"
+PROMPT_END="
+╰ %B$%b "
+
+
+PROMPT="${PROMPT_START} ${PROMPT_END}"
 RPS1="${return_code}"
 
 ZSH_THEME_GIT_PROMPT_PREFIX="%{$fg[yellow]%}‹"
 ZSH_THEME_GIT_PROMPT_SUFFIX="› %{$reset_color%}"
+ZSH_THEME_GIT_PROMPT_CLEAN="✔"
+ZSH_THEME_GIT_PROMPT_DIRTY="✗"
+
+source $ZSH/lib/async-git-prompt.zsh
```

Comment:
No need to `source` this. Everything in `$ZSH/lib` is always sourced by default. (And `$ZSH_CUSTOM` may override it.)